test/extended/oauth/requestheaders.go

```diff
@@ -234,6 +238,19 @@ var _ = g.Describe("[Serial] [sig-auth][Feature:OAuthServer] [RequestHeaders] [I
 	})
 })
 
+func gatherPostMortem(oc *exutil.CLI) {
+	authn, err := oc.AdminConfigClient().ConfigV1().ClusterOperators().Get(context.Background(), "authentication", metav1.GetOptions{})
+	if err != nil {
+		e2e.Logf("Error getting authentication operator: %v\n", err)
+	}
+	e2e.Logf("Authentication %v\n", authn)
+	routerCA, err := oc.AdminKubeClient().CoreV1().ConfigMaps("openshift-config-managed").Get(context.Background(), "default-ingress-cert", metav1.GetOptions{})
```

Comment:
I'm not sure how retrieving this certificate helps, it's more or less a constant and if it were broken, many other things would be broken as well.

Maybe you may want to check:
a) the actual status of the oauth-openshift deployment
b) current config in the openshift-authentication NS
c) the logs of all the oauth-openshift deployment pods